DataParsers/XmlWeatherDataParsingStrategy.cs

```diff
@@ -0,0 +1,19 @@
+using System.Xml.Linq;
+using Models;
+namespace DataParsers
+{
+    public class XmlWeatherDataParsingStrategy : IWeatherDataParsingStrategy
+    {
+        public WeatherData ParseData(string dataFile)
+        {
+            string configXml = File.ReadAllText(dataFile);
+            XElement xml = XElement.Parse(configXml);
+            return new WeatherData
+            {
+                Location = xml.Element("Location").Value,
```

Comment:
always keep validation and error handling in mind,  what if there wasn't a location tag ?